ui/apps/platform/src/Containers/NetworkGraph/namespace/NamespaceSideBar.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+    Badge,
+    Flex,
+    FlexItem,
+    Tab,
+    TabContent,
+    Tabs,
+    TabTitleText,
+    Text,
+    TextContent,
+    TextVariants,
+} from '@patternfly/react-core';
+
+import useTabs from 'hooks/patternfly/useTabs';
+import NamespaceDeployments from './NamespaceDeployments';
+
+function NamespaceSideBar() {
+    const { activeKeyTab, onSelectTab } = useTabs({
+        defaultTab: 'Deployments',
+    });
+
+    return (
+        <Flex direction={{ default: 'column' }} flex={{ default: 'flex_1' }} className="pf-u-h-100">
+            <Flex direction={{ default: 'row' }} className="pf-u-p-md pf-u-mb-0">
+                <FlexItem>
+                    <Badge style={{ backgroundColor: 'rgb(32,79,23)' }}>NS</Badge>
+                </FlexItem>
+                <FlexItem>
+                    <TextContent>
+                        <Text component={TextVariants.h1} className="pf-u-font-size-xl">
+                            stackrox
+                        </Text>
+                    </TextContent>
+                    <TextContent>
+                        <Text
+                            component={TextVariants.h2}
+                            className="pf-u-font-size-sm pf-u-color-200"
+                        >
+                            in &quot;remote&quot;
```

Comment:
@alanonthegit 
The way we have to code these quotes reminded me to ask:

Should we be emphasizing the k8s hierarchy (cluster or cluster/namespace) with something besides quotes?
Like **bold** and/or *italic*

Quotes looks a little like an afterthought, and if we internalize our app, neither locals use different symbols for quote marks, so the meaning is more obscure.

Reply:
On that note, is it even necessary to have this if we have the breadcrumbs up top?